File: Kata.GameOfLife/Cell.cs
using System;

namespace Kata.GameOfLife
{
    public class Cell   : IEquatable<Cell>
    {
        public Cell(int x, int y)
        {
            X = x;
            Y = y;
        }

        public int X { get; set; }

        public int Y { get; set; }
        public bool Equals(Cell other)
        {
            if (ReferenceEquals(this, other)) return true;

            if (this.X == other.X && this.Y == other.Y)
            {
                return true;
            }

            return false;
        }
    }
}

File: Kata.GameOfLife.Python/cell.py
class Cell:
  def __init__(self, x, y):
    self.x = x
    self.y = y

  def __str__(self):
    return "x: {0}, y: {1}".format(self.x, self.y)

  def equals(self, otherCell):
    return self.x == otherCell.x and self.y == otherCell.y